### DISCUSSION TITLE
"Someday, we will tape out chips". I need firm date.

### DISCUSSION THREAD

**Discussion by MotorCityCobra** [tinygrad/tinygrad#2464] [Q&A]
Can tinygrad take the AI chip market from Nvidia because the tinygrad chip will be for the tinygrad framework?
There are lots of chips and people promising to make chips. Will a chip made specifically for this framework be optimal? How?
Is there anything happening on this front?

Has George talked about the road map to tinygrad chips in a video? The GitHub says, "Someday, we will tape out chips".
Is this something I can get started working on? Where would I begin?
Will tinygrad beat Nvidia because we'll have framework integrated from silicon to the last layer?
Many AI chip startups have been announced in recent years. Is Nvidia still dominating the market with a 1000% profit margin because the AI chip makers didn't have a framework?

**Comment by QuentinWach** [comment]
Same question here. I just started looking into it and building something. But it is strange to see no one really talking about it. And those who talk about custom chips talk about GPUs even though the goal is clearly to create a non-Turing complete ASIC made specifically to accelerate tinygrad with its instruction set.
